src/components/src/ImageContainer/image.container.scss

```diff
@@ -0,0 +1,3 @@
+picture {
+    width: 100%;
+}
```

Comment:
One empty line should be at the end of each file. We have a lint rule for code for this but apparently it doesn't work for scss files :)